src/modules/private-share-folder/private-sharring.usecase.spec.ts

```diff
@@ -172,8 +167,15 @@ describe('PrivateSharingUseCase', () => {
         order,
       );
     });
+  });
+
+  describe('Manage private shared folder roles', () => {
+    const userUuid = v4();
+    const privateFolderId = v4();
+    const roleUuid = v4();
+    const owner = user;
 
-    it('should grant privileges if the owner is indeed the owner of the folder', async () => {
+    it('Grant privileges by owner', async () => {
```

Comment:
Hey, check this legend, as an example: 
_When privileges are granted by the owner, then the invited user gets role_